src/groovy/org/transmartproject/export/GENExporter.groovy

```diff
@@ -89,13 +92,13 @@ class GENExporter implements HighDimTabularResultExporter {
                 out << ' '
                 out << row.a2
                 
-                for (/*SnpLzCell*/ Object cell: row) { 
+                for (double[] cell: row) {
                     out << ' '
-                    out << cell.probabilityA1A1
+                    out << cell[0] // probabilityA1A1
                     out << ' '
-                    out << cell.probabilityA1A2
+                    out << cell[1] // probabilityA1A2
                     out << ' '
-                    out << cell.probabilityA2A2
+                    out << cell[2] // probabilityA2A2
```

Comment:
Yes, the current behavior for a `double[]` would be to convert it to String. 

The rest-api needs to be told in some way what to do with the value if just converting to `String` or `Double` is not the right thing. Currently there is only support for objects with properties, including maps, using the `MultiValueProjection` marker interface. We could extend that to `Iterables` (dunno if we'd need to special case primitive arrays, for Java callers we probably would), but that would require e.g. an `IterableProjection` interface. We could also use `0`, `1`, `2` as the property "names" to use, but that would result in nondescriptive column names in the R output.